templates/base.html

```diff
@@ -14,15 +14,20 @@
   <meta http-equiv="Content-Type" content="application/xhtml+xml; charset=utf-8" />
   <meta name="keywords" content="{% block meta-keywords %}{% endblock %}">
   <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.0/css/bulma.min.css">
-{% comment %}
-{#
-We're pinning vocabulary v2020.11.1 because v2020.11.2 had a change that
-broke headers for this site. So far, I've been unable to find any
-documentation about it other than the warning in the release notes that
-there was a breaking change.
-#}
-{% endcomment %}
-  <link rel="stylesheet" href="https://unpkg.com/@creativecommons/vocabulary@v2020.11.1/css/vocabulary.css">
+  <link
+    rel="stylesheet"
+    href="https://unpkg.com/@creativecommons/fonts@2020.9.3/css/fonts.css"
+  />
+  <link
+    rel="stylesheet"
+    href="https://unpkg.com/@creativecommons/vocabulary@v2020.11.3/css/vocabulary.css"
+  />
+  <!-- Import Vue JS via CDN -->
+  <script src="https://unpkg.com/vue@next"></script>
+  <!-- Import Axios via CDN. Needed for the Global Header Component -->
+  <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+  <!-- Import the Component Library via CDN -->
+  <script src="https://unpkg.com/@creativecommons/cc-global-components@0.1.1/dist/cc-globals.min.js"></script>
```

Comment:
Since we may make some backward-compatible changes in the near future, it might save time to specify a version range here, such as

```suggestion
  <script src="https://unpkg.com/@creativecommons/cc-global-components@0.x.x/dist/cc-globals.min.js"></script>
```